frontend/src/views/RouteFormView.vue

```diff
@@ -147,21 +144,32 @@ export default {
     ...mapActions(namespaces.passengers, [apiActions.fetchPassengers]),
     handleSubmit() {
       this.validateForm();
-      this.isSubmitted = true;
 
-      if (!this.errors.length) {
-        this.route.id = uuidv4();
+      if (!Object.keys(this.errors).length) {
         this[actions.SUBMIT]({ form: this.route });
         this.route = { ...defaultFormState };
-        this.isSubmitted = false;
       }
     },
 
     validateForm() {
       const { route } = this;
+
+      const makeErrorsPartial = makeErrors(this.$t.bind(this));
+
       this.errors = Object.keys(route)
         .filter(isErroring(route))
-        .map(makeErrorMessage(this.$t.bind(this)));
+        .reduce(makeErrorsPartial, {});
+
+      const { startMileage, endMileage } = this.route;
+
+      if (
+        !!startMileage
+        && !!endMileage
+        && parseInt(startMileage, 10) >= parseInt(endMileage, 10)
+      ) {
+        this.errors.startMileage = this.$t('common.start_mileage_error');
```

Comment:
Errors are not translated if I switch language after validating form. We can live with that